Gemfile

```diff
@@ -120,6 +120,8 @@ group :development, :test do
 end
 
 group :development do
+  # Process manager for applications with multiple components
+  gem 'foreman'
```

Comment:
This is an alternative for Docker setup, use this to start all services in Procfile